src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -74,11 +89,24 @@ public void setProxy(SaltStackProxySettings settings) {
      */
     public SaltStackToken login(String username, String password)
             throws SaltStackException {
+        return login(username, password, "auto");
```

Comment:
I would rather suggest to avoid strings and document them in this case, as this param is actually SaltStack method to connect, perhaps?

```java
// Documentation, describing what does that mean
public class SaltStackAPIConnectionConstants {
    public static final AUTO = "auto";
    public static final OTHER_POSSIBLE_CONNECTION = "other_possible_connection";
}

...

    return login(username, password, SaltStackAPIConnectionConstants.AUTO);
```